ads/colombia.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
(Because I made the same mistake) You need to make all new files with a 2016 copyright notice, including columbia.md.